client/signup/steps/site-information/index.jsx

```diff
@@ -0,0 +1,241 @@
+									<FormFieldset>
+										<FormLabel htmlFor="email">
+											{ translate( 'Contact Email' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'Does your business have an email address?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTextInput
+											id="email"
+											name="email"
+											isValid={ this.state.isEmailValid }
+											isError={ this.state.isEmailValid }
+											placeholder={ 'E.g. email@domain.com' }
+											onChange={ this.handleInputChange }
+											value={ this.state.email }
+										/>
+										{ this.state.shouldShowEmailError && (
+											<FormInputValidation
+												isError={ this.state.shouldShowEmailError }
+												text={ translate( 'Please provide a valid email address.' ) }
+											/>
+										) }
+									</FormFieldset>
+									<FormFieldset>
+										<FormLabel htmlFor="phone">
+											{ translate( 'Phone number' ) }
+											<InfoPopover className="site-information__info-popover" position="top">
+												{ translate( 'How can people contact you?' ) }
+											</InfoPopover>
+										</FormLabel>
+										<FormTextInput
+											id="phone"
+											name="phone"
+											placeholder={ translate( 'E.g. (613) 425-0183' ) }
+											onChange={ this.handleInputChange }
+											value={ this.state.phone }
+										/>
+									</FormFieldset>
+								</Fragment>
+							) }
+							<div className="site-information__submit-wrapper">
+								<Button primary={ true } type="submit">
+									{ translate( 'Continue' ) }
+								</Button>
+							</div>
+						</Card>
+					</form>
+				</div>
+			</div>
+		);
+	}
+
+	render() {
+		const { flowName, positionInFlow, signupProgress, stepName, translate } = this.props;
+
+		const headerText = translate( 'Almost done, just a few more things.' );
+		const subHeaderText = translate( "We'll add this information to your new website." );
+
+		return (
+			<StepWrapper
+				flowName={ flowName }
+				stepName={ stepName }
+				positionInFlow={ positionInFlow }
+				headerText={ headerText }
+				fallbackHeaderText={ headerText }
+				subHeaderText={ subHeaderText }
+				fallbackSubHeaderText={ subHeaderText }
+				signupProgress={ signupProgress }
+				stepContent={ this.renderContent() }
+			/>
+		);
+	}
+}
+
+/**
+ * Returns translated label and placeholder for the name field depending on the site type
+ *
+ * @param   {String} siteType E.g,'business', 'blog'
+ * @returns {Object} label and placeholder object
+ */
+function getSiteNameText( siteType ) {
+	//TODO: After site-type component is merged, fetch segment names from lib/site-type.js
+	switch ( siteType ) {
+		case 'business':
+			return {
+				label: i18n.translate( 'Business name' ),
+				placeholder: i18n.translate( 'eg. Google, Automattic, AirBnb' ),
+			};
+		case 'blogger':
+			return {
+				label: i18n.translate( 'Blog name' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **75** times:
`translate( 'Blog Name' )` *ES Score: 11*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/onboarding-step-site-information-component)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).